src/server/api/controllers/events.controller.js

```diff
@@ -17,14 +18,16 @@ const createEvent = async (body) => {
 
 const getEvents = async () => {
   try {
-    return await knex('events').select(
-      'events.id',
-      'events.event_type',
-      'event_date',
-      'venue',
-      'max_participants',
-      'description',
-    );
+    return await knex('events')
+      .select(
+        'events.id',
+        'events.event_type',
+        'event_date',
+        'venue',
+        'max_participants',
+        'description',
```

Comment:
If you have time, you could fix these to properly format property names. it should be `eventType`, `eventDate`, etc.

Not necessary, just a nice thing to have.